File: Backend/full-stack-chat-app-backend/Models/Message.cs


namespace full_stack_chat_app_backend.Models
{
    public class Message
    {
         public string SenderDisplayName {get; set;} = string.Empty;
         public string MessageContent {get; set;} = string.Empty;
         public Message(string SenderDisplayName,string MessageContent){
            this.SenderDisplayName = SenderDisplayName;
            this.MessageContent = MessageContent;
         }
    }
}

File: Client/src/Components/Message/Message.js
import "./Message.css"
function Message(props) {
    return (
    <div>
        <p className={props.usersMessage ? "messageUserName":"messageOtherName"}>@{props.senderDisplayName}</p>
        <p className = {props.usersMessage ? "messageUser":"messageOther"}>
        {props.messageContent}
    </p>
    </div>);
}

export default Message;